docs/book/src/introduction/sway_quickstart.md

```diff
@@ -202,6 +202,26 @@ Note the contract ID — you will need it if you want to build out a frontend to
 
 If `forc-deploy` emits a `no such subcommand` error, try to install it with [fuelup](../introduction/installation.md) or [manually](https://github.com/FuelLabs/sway/releases).
 
+## Deploying the contract with Salt
+
+You can pass a Salt using the `--salt` flag in your `forc deploy` command to redeploy a contract that has been deployed already. This is useful for multiple deployments of the same contract.
+
+To add a Salt to your ContractID while deployment, you can run the following command:
+
+```bash
+forc deploy --salt <Salt> --node-url <key> --gas-price 1
+```
+
+### Randomised Salt
+
+Salt is 0x00000000000000000000000000000000 by default. If you do not want to pass a salt manually, you can also use the --random-salt flag to randomise your salt as follows:
+
+```bash
+forc deploy --random-salt --node-url <key> --gas-price 1
+```
+
+> Note: To be able to use the `forc deploy` and `forc run` commands, you require the `forc client` plugin that lets you interact with a Fuel node. You can read more about `forc client` [here](https://fuellabs.github.io/sway/v0.35.2/book/forc/plugins/forc_client/index.html).
```

Comment:
We should not point to a published version of the book. Instead, a relative path here is better.